src/utils/stringUtils.js

```diff
@@ -0,0 +1,3 @@
+export const asteriskRepeat = (termLength) => {
```

Comment:
Did you test this code before creating a pull request?

Reply:
Yes. It works even now!